app/javascript/components/student/TracksList.jsx

```diff
@@ -42,6 +42,7 @@ export function TracksList({ statusOptions, tagOptions, ...props }) {
       <div className="search-bar">
         <div className="md-container">
           <Search dispatch={dispatch} />
+          <TagsFilter dispatch={dispatch} options={tagOptions} />
         </div>
       </div>
       <div className="md-container">
```

Comment:
```suggestion
      <div className="md-container">
```

Should probably be a `section`